### PR TITLE
Add css loader

### PR DESCRIPTION
**Pull request by ACEof**
Added css-loader to project.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -18,15 +18,20 @@
   },
   "devDependencies": {
     "babel-core": "^6.26.0",
+    "babel-eslint": "^8.2.3",
     "babel-loader": "^7.1.4",
     "babel-preset-env": "^1.6.1",
     "babel-preset-react": "^6.24.1",
+    "css-loader": "^0.28.11",
```
Comment (abtv) [inline package.json]: what is it for?

File: package.json [hunk 2]
```diff
@@ -18,15 +18,20 @@
...
     "eslint-config-airbnb": "^16.1.0",
-    "eslint-plugin-import": "^2.11.0",
+    "eslint-plugin-import": "^2.12.0",
     "eslint-plugin-jsx-a11y": "^6.0.3",
-    "eslint-plugin-react": "^7.7.0",
+    "eslint-plugin-react": "^7.8.2",
     "html-webpack-plugin": "^3.1.0",
     "jest": "^22.4.3",
+    "mini-css-extract-plugin": "^0.4.0",
+    "style-loader": "^0.21.0",
```
Comment (abtv) [inline package.json]: what is it for?

**Comment by abtv** [comment]
Why is the pull request called `Add css loader`?

File: tests/lexers.test.js [hunk 3]
```diff
@@ -0,0 +1,87 @@
+impot { parseTockens } from '../src/componets/lexer'
+test ('Checking the search function for finding tokens: let a, let b ', () => {
+  expect(parseTockens('let a
+                        a = 5
+                        let b')).toEqual({
```
Comment (abtv) [inline tests/lexers.test.js]: I expect you to fix it

Comment (abtv) [reply tests/lexers.test.js]: we discussed it should be array of tokens, not a map

Comment (ACEof) [reply tests/lexers.test.js]: This is old version. She got into this PR because of my carelessness.

**Pull request closed without merging**